src/figuresClasses.ts

```diff
@@ -1,19 +1,100 @@
 export interface Figure {
+  shape: string;
+  color: string;
+  sizes: number[];
 
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape: string;
 
-}
+  sizes: number[];
+
+  constructor(
+    public color: string,
+    ...args: number[]
+  ) {
+    this.shape = 'triangle';
+    this.color = color;
```

Comment:
You already done this, when call this field in constructor. This is code duplication